.github/workflows/linterns.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters

```

Comment:
- [ ] Great job adding linters to your PR, however, they should be green, kindly consider fixing all the linter errors so they pass

Reply:
![lintern1](https://user-images.githubusercontent.com/93407216/185234755-142f63d1-018d-4bbc-838f-d15c710d0f5e.png)
![lintern2](https://user-images.githubusercontent.com/93407216/185234760-300143bb-d9a6-421c-9e09-1705561c5944.png)

Reply:
Hello @Ismailco ! thanks for your review. I'm not sure what are you asking for. Because I see the test all in green.
Would you be so kind to explain to me what I should fix?